draft-ietf-httpbis-priority.md

```diff
@@ -402,10 +402,27 @@ Prioritized Stream ID:
 Priority Field Value:
 : The priority update value in ASCII text, encoded using Structured Fields.
 
-The Prioritized Stream ID MUST be within the stream limit. If a
-server receives a PRIORITY_UPDATE with a Prioritized Stream ID that is beyond
-the stream limits, this SHOULD be treated as a connection error of type
-PROTOCOL_ERROR.
+When the PRIORITY_UPDATE frame applies to a request stream, a client SHOULD
+provide a Prioritized Stream ID that refers to a stream in the "open",
+"half-closed (local)", or "idle" state. Frames where the Prioritized Stream ID
+refers to a stream in the "half-closed (remote)" or "closed" state can be
+discarded by the server upon receipt. The number of streams which have been
```

Comment:
```suggestion
"half-closed (local)", or "idle" state. The server can discard frames where the
Prioritized Stream ID refers to a stream in the "half-closed (local)" or "closed"
state can be discarded upon receipt. The number of streams which have been
```
"Half-closed (local)" on the server is "half-closed (remote)" on the client.  The switch of perspective can be signaled by making the server the subject of the sentence.